src/pages/Registration.tsx

```diff
@@ -143,14 +141,10 @@ const Registration = () => {
 
     return (
         <FullPageDialog>
-            <Card
-                raised={false}
-                sx={{
-                    background: 'transparent',
-                }}
-            >
-                <CardContent
+            <>
```

Comment:
Any component using `FullPageDialog` will need to be a fragment, especially if its content expands beyond the viewport.